docs/subnets/hello-world-precompile-tutorial.md

```diff
@@ -118,6 +118,15 @@ provide access to the EVM state including the ability to modify balances and rea
 This way we can provide even more customization of the EVM through Stateful Precompiles than we can
 with the original precompile interface!
 
+### AllowList
+
+The AllowList enables a precompile to enforce permissions on addresses. The AllowList is not a contract
+itself, but a helper structure to provide a control mechanism for wrapping contracts.
+It provides an `AllowListConfig` to the precompile so that it can take an initial configuration
+from genesis/upgrade. It also provides 4 functions to set/read the permissions. In this tutorial, we
+used `AllowList` to provide permission control to the `HelloWorld` precompile.
+You can find more information about the AllowList interface [here](../subnets/customize-a-subnet.md#allowlist).
```

Comment:
should the link be `../subnets/customize-a-subnet.md#allowlist-interface`? There is no `../subnets/customize-a-subnet.md#allowlist` in the doc.

Reply:
yep true good catch.